readthedocs/api/v3/serializers.py

```diff
@@ -427,6 +435,41 @@ class Meta:
         )
 
 
+class ProjectUpdateSerializer(FlexFieldsModelSerializer):
+
+    """Serializer used to modify a Project once imported."""
+
+    repository = RepositorySerializer(source='*')
```

Comment:
What is `source='*'`? It should likely have a comment.

Reply:
`*` it's standard for DRF but has a special meaning. It means that the whole object will be passed to the `RepositorySerializer`, not just a specific field of it.

(see docs https://www.django-rest-framework.org/api-guide/fields/#source)